testsuite/tests/conftest.py

```diff
@@ -176,13 +177,28 @@ def pytest_collection_modifyitems(session, config, items):
     https://docs.pytest.org/en/stable/usage.html
     """
 
+    sso_only_opt = config.option.sso_only
+
+    selected_fixtures = ["rhsso_setup"]
+    selected = []
+    deselected = []
+
     for item in items:
         for marker in item.iter_markers(name="issue"):
             issue = marker.args[0]
             issue_id = issue.rstrip("/").split("/")[-1]
             item.user_properties.append(("issue", issue))
             item.user_properties.append(("issue-id", issue_id))
 
+        if sso_only_opt and not any(fixture in selected_fixtures for fixture in item.fixturenames):
```

Comment:
we need to deselect performance tests from sso tag, there are 4 tests from performance